sycl/include/sycl/ext/intel/experimental/esimd/detail/memory_intrin.hpp

```diff
@@ -519,8 +499,159 @@ __esimd_raw_send_store(uint8_t modifier, uint8_t execSize,
                        uint8_t numSrc0, uint8_t sfid, uint32_t exDesc,
                        uint32_t msgDesc,
                        __SEIEED::vector_type_t<Ty1, N1> msgSrc0);
+
 #ifndef __SYCL_DEVICE_ONLY__
 
+/// ESIMD_CPU Emulation support using esimd_cpu plugin
+
+#define __SYCL_EXPLICIT_SIMD_PLUGIN__
+
+// Header files required for accessing CM-managed memory - Surface,
+// buffer, etc
+namespace cm_support {
+#include <CL/cm_rt.h>
+} // namespace cm_support
+
+#include <CL/sycl/backend_types.hpp>
+#include <CL/sycl/detail/pi.hpp>
+#include <sycl/ext/intel/experimental/esimd/detail/atomic_intrin.hpp>
+#include <sycl/include/CL/sycl/INTEL/esimd/detail/emu/esimdcpu_device_interface.hpp>
+
+namespace raw_send {
```

Comment:
It seem this namespace declared in the "globally"(not in the namespace user isn't supposed to use).  If so, suggest moving it to `cl::sycl::__detail` namespace.

Reply:
Will move to suggested name space.

Reply:
`raw_send` namespace is put under `sycl::ext::intel::experimental::esimd::detail` (aliased with `__SEIEED`)